### ISSUE TITLE
Sponsorship: Add operator cupid

### ISSUE THREAD

**Issue by ghenry** [DrHyde/perl-modules-Number-Phone#59]
Hi,

Could we sponsor adding the operators cupid?

Also, how accurate is operator_ported

Thanks.

**Comment by DrHyde** [comment]
Yes, by all means I'd be happy to add more data for operators, although I'd want to at least try to do so in a way that will work in multiple countries. If you've got code for it then I'd be happy to take a pull request, otherwise I'll add it to my to-do list.

operator_ported is only a stub that returns undef, as I haven't been able to find any freely-available data sources.

**Comment by DrHyde** [comment]
It looks like someone has written a Number::Phone::RO for Romania, and it knows about number portability via operator_ported(). Don't hold your breath for OFCOM to release the data to make that possible for UK numbers though!

**Comment by ghenry** [comment]
I'm a bit behind on my emails. Will be in touch soon.

Thanks.

--
Kind Regards,

Gavin Henry.
Managing Director.

D +44 (0) 330 44 55 007
T +44 (0) 330 44 50 000
M +44 (0) 7930 323266
F +44 (0) 1224 824887
E ghenry@suretec.co.uk

Open Source. Open Solutions(tm).

http://www.suretecsystems.com/

Winner of the Best Business ITSP (Medium Enterprise) 2016!
http://www.surevoip.co.uk/2016-best-provider

Suretec Systems is a limited company registered in Scotland. Registered
number: SC258005. Registered office: 24 Cormack Park, Rothienorman,
Inverurie,
Aberdeenshire, AB51 8GL.

Subject to disclaimer at http://www.suretecgroup.com/disclaimer.html

Do you know we have our own VoIP provider called SureVoIP®? See
http://www.surevoip.co.uk

OpenPGP (GPG/PGP) Public Key: 0x8CFBA8E6 - Import from hkp://subkeys.pgp.net
or http://www.suretecgroup.com/0x8CFBA8E6.gpg
